tools/makecsr/makecsr.go

```diff
@@ -39,20 +18,59 @@ import (
 	"github.com/mozilla-services/autograph/crypto11"
 )
 
+var (
+	allowedSigAlgs = map[string]x509.SignatureAlgorithm{
+		"SHA256WithRSA":   x509.SHA256WithRSA,
+		"SHA384WithRSA":   x509.SHA384WithRSA,
```

Comment:
SHA384WithRSA doesn't work with GCP KMS. Can we add a note here to remove this when AWS support is removed? I also wonder if it's worth checking for this explicitly further down so we can offer a very clear error message.

Reply:
Added. PTAL